### PR TITLE
[4.0] Fix custom-select styling in  frontend template

### PR DESCRIPTION
**Pull request by ciar4n**
Pull Request for Issue #18623 .

### Summary of Changes
Removes the native dropdown on custom select fields in frontend. Appears to be a known bootstrap bug (https://github.com/twbs/bootstrap/issues/16201). 

In the original issue there is two screenshots of the issue. This should fix the screen by @franz-wohlkoenig. I am unable to replicate the screenshot by @PhilETaylor 

### Testing Instructions
Apply PR and attempt to edit article in frontend (Publishing tab).

### CODE REVIEW

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on f15a8a6a177cbfe721715c79864bc7018b26ef87<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18663">issues.joomla.org/tracker/joomla-cms/18663</a>.</sub>

**Comment by wilsonge** [comment]
@ciar4n sorry can you fix conflicts again? I've apparently screwed by sass gem trying to update it just now :(

**Comment by ciar4n** [comment]
@wilsonge np.. conflicts fixed.

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on f8efb9d92bb64125c37e9cb811640a6e389d23ba<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18663">issues.joomla.org/tracker/joomla-cms/18663</a>.</sub>

File: templates/cassiopeia/css/template.css [hunk 1]
```diff
@@ -8993,7 +8998,7 @@ joomla-alert {
...
-        grid-template-areas: ". head head head head ." ". banner banner banner banner ." ". main main main main ." ". top-a top-a top-a top-a ." ". top-b top-b top-b top-b ." ". bot-a bot-a bot-a bot-a ." ". bot-b bot-b bot-b bot-b ." ". footer footer footer footer ." ". debug debug debug debug ."; } }
+        grid-template-areas: ". head head head head ."
 ". banner banner banner banner ."
 ". main main main main ."
 ". top-a top-a top-a top-a ."
 ". top-b top-b top-b top-b ."
 ". bot-a bot-a bot-a bot-a ."
 ". bot-b bot-b bot-b bot-b ."
 ". footer footer footer footer ."
 ". debug debug debug debug ."; } }
```
Comment (dgrammatiko) [inline templates/cassiopeia/css/template.css:9001]: @C-Lodder @ciar4n @wilsonge we still need a solution here, this inconsistency between windows and macs is really annoying

Comment (C-Lodder) [reply templates/cassiopeia/css/template.css:9001]: There's already a solution. I believe we went through it in London. Just requires setting changes for Git.

**Comment by C-Lodder** [comment]
@ciar4n this can be closed as has already been fixed

**Pull request closed without merging**